{{cookiecutter.project_slug}}/pytest.ini

```
@@ -1,5 +1,5 @@
 [pytest]
-DJANGO_SETTINGS_MODULE=config.settings.test
+addopts = --ds=config.test
```

I think it should be:

```suggestion
addopts = --ds=config.settings.test
```